Switch/SWCoreWindowService.m

```diff
@@ -404,4 +399,40 @@ - (void)private_hideInterface;
     [self.interface shouldShowInterface:false];
 }
 
+- (void)private_raiseWindow;
+{
+    SWWindow *selectedWindow = self.stateMachine.selectedWindow;
+    if (!selectedWindow) { return; }
+    
+    @weakify(self);
+    [[SWAccessibilityService sharedService] raiseWindow:selectedWindow completion:^(NSError *error) {
+        @strongify(self);
+        if (error) {
+            SWLog(@"Failed to raise window group %@: %@", selectedWindow, error);
+        }
+        
+        NSOrderedSet *windowList = self.stateMachine.windowList;
+        NSUInteger selectedIndex = [windowList indexOfObject:selectedWindow];
+
+        if (selectedIndex == 0) {
+            // If the selected index was 0, this action won't change the window order so this code must replay the last update event.
+            [self.stateMachine updateWindowList:self.stateMachine.windowList];
```

Comment:
In an ideal world, this will break when #122 gets fixed, as `activeApplication` should become an immutable data property.

Unfortunately, `refreshWindowList` won't dispatch a notification if there is only a change in active application, so that'll have to happen later. Sorry, future me.